sql/src/lib.rs

```diff
@@ -69,11 +113,70 @@ impl QueryBuilder for SQLiteQueryBuilder {
         Ok(())
     }
 
-    fn push_bind_param(&mut self) {
-        self.push_sql("?");
+    fn push_typed_value(&mut self, value: &TypedValue) -> BuildQueryResult {
+        use TypedValue::*;
+        match value {
```

Comment:
I think clippy suggest `match *value`, but I don't quite understand why.  @victorporof and I are trying to standardize on the former, assuming that clippy "knows something"; will you join us?